{"name": "test_add[case_info0]", "status": "passed", "attachments": [{"name": "stdout", "source": "4b51f262-5ff3-4b9e-a6a9-25d41cd76803-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "5c281fa6-9b20-4123-9481-2dbd5ee8f676-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "case_info", "value": "{'case_id': 1, 'module': '登录', 'title': 'login failed', 'data': '{\"username\":\"23\", \"password\": \"1234\"}', 'expected': 'error'}"}], "start": 1643798343472, "stop": 1643798343472, "uuid": "0910f00a-2cff-4b29-8381-8b6906e741bd", "historyId": "c5aeb5b1622992d1487025b9c5b6e022", "testCaseId": "a4b9f2c635a80802701f818745729390", "fullName": "testcase.test_web.test_login.TestLogin#test_add", "labels": [{"name": "tag", "value": "add"}, {"name": "parentSuite", "value": "testcase.test_web"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-9072V9N"}, {"name": "thread", "value": "5888-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_web.test_login"}]}